spec/resizingSpec.js

```diff
@@ -4,7 +4,7 @@ describe("Resizing Dragdealer", function() {
     this.addMatchers(matchers);
   });
 
-  it("should translate handle position after window resize", function() {
+  exceptInPhantomJsIt("should translate handle position after window resize", function() {
```

Comment:
Kudos for the effort, but it's pretty nasty. I looked into why this test is failing and it turns out you can't fake an event using `window.dispatchEvent(new Event('resize'));` in PhantomJS: https://github.com/ariya/phantomjs/issues/11289

Give we already had this [big `if` for IE](https://github.com/flosell/dragdealer/blob/tests-without-sauce-labs-on-pull-requests/spec/resizingSpec.js#L15-L22) where we fall back on [calling the resize handler by hand](https://github.com/flosell/dragdealer/blob/tests-without-sauce-labs-on-pull-requests/spec/resizingSpec.js#L19), I say we remove the createEvent code altogether and stick with just calling the resize handler, since it's outside of the scope of that test anyway (no point in testing the browser's native resize behavior).

Do that you can remove the `exceptInPhantomJsIt` handler altogether and the tests will run identically in PhantomJS and SauceLabs.